packages/runtime/src/account.ts

```diff
@@ -184,6 +184,58 @@ export class StoreAccount implements StoreAccountI {
     return asset.definitions;
   }
 
+  /**
+   * Modifies Asset fields
+   * @param assetId Asset Index
+   * @param fields Fields for modification
+   */
+  modifyAsset (assetId: number, fields: AssetModFields): void {
+    const asset = this.getAssetDef(assetId);
+    if (asset === undefined) {
+      throw new TealError(ERRORS.ASA.ASSET_NOT_FOUND, { assetId: assetId });
+    }
+    // check for blank fields
+    if ((fields.reserve && asset.reserve === "") || (fields.freeze && asset.freeze === "") ||
+      (fields.clawback && asset.clawback === "")) {
+      throw new Error("Cannot reset a blank address");
+    }
+
+    asset.manager = fields.manager;
+    asset.reserve = fields.reserve;
+    asset.freeze = fields.freeze;
+    asset.clawback = fields.clawback;
+  }
+
+  /**
+   * Freeze asset
+   * @param assetId Asset Index
+   */
+  freezeAsset (assetId: number, state: boolean): void {
+    const holding = this.assets.get(assetId);
```

Comment:
```suggestion
    const account = this.assets.get(assetId);
```

Reply:
I don't think holding is the right word to be used here. We are getting an account, host account. So other good name would be `host`.

Reply:
Here we are getting assetHolding from local state of a account using
This.assets.get(), assets is a map which stores AssetHolding Type as value.

Should I change it to something else ?

Reply:
no, it's fine, I read it too fast.